server/src/main/java/com/tinyfs/handler/ClientHandler.java

```diff
@@ -23,7 +21,7 @@
 import com.tinyfs.validation.ClientRegistrationRequestValidator;
 
 @Component
-public class ClientHandler extends TextWebSocketHandler {
+public class ClientHandler extends BinaryWebSocketHandler {
```

Comment:
It was pretty unclear to me too. As I said earlier, I didn't know that it was part of the websocket standard until I read the client websocket implementation